torch_geometric/sampler/utils.py

```diff
@@ -27,16 +28,32 @@ def to_csc(
     perm: Optional[Tensor] = None
 
     if hasattr(data, 'adj'):
+        if src_node_time is not None:
+            raise NotImplementedError("Temporal sampling via 'SparseTensor' "
+                                      "format not yet supported")
         colptr, row, _ = data.adj.csc()
 
     elif hasattr(data, 'adj_t'):
+        if src_node_time is not None:
```

Comment:
cc @mananshah99: I don't have any good idea how we can support this. `SparseTensor` will always automatically sort by column index when not explicitly initialized via `is_sorted=True`.

Reply:
At worst, we can convert to COO and back to CSC, but that's not very satisfying. Maybe it's a good stop-gap?

Reply:
👍